backend/src/utils/semantic_layer_builder.py

```diff
@@ -0,0 +1,78 @@
+# NOT NEEDED CURRENTLY AS THE RETURNED VALUE OF GRAPH SCHEMA IS STORED STATICALLY AS A JINJA TEMPLATE
+
+from src.utils import call_model
+from src.utils.graph_db_utils import execute_query
+import logging
+from src.prompts import PromptEngine
+import json
+import ast
+
+logger = logging.getLogger(__name__)
+
+engine = PromptEngine()
+
+def get_semantic_layer():
+    finalised_graph_structure = {'nodes':{}, 'properties':{}}
+
+    neo4j_graph_why_prompt = engine.load_prompt("neo4j-graph-why")
+
+    relationship_query = engine.load_prompt("relationships-query")
+
+    node_query = engine.load_prompt("nodes-query")
+
+    relationship_property_query = engine.load_prompt("relationship-property-cypher-query")
```

Comment:
For comment above: add in an arguement here and can then just load the same prompt eachtime with a different arguement to differentiate